protocol-concepts/governance/verification.md

```diff
@@ -28,4 +42,19 @@ exchangeId = keccak256(
+<table><thead><tr><th width="176.33333333333331">Exchange</th><th>ID</th><th>Explanation</th></tr></thead><tbody>
+<tr><td>cUSD/CELO</td><td>0x3135b662c38265d0655177091f1b647b4fef511103d06c016efdf18b46930d2c</td><td><code>keccak256(abi.encodePacked("cUSD", "CELO", "ConstantProduct"))</code></td></tr>
+<tr><td>cEUR/CELO</td><td>0xb73ffc6b5123de3c8e460490543ab93a3be7d70824f1666343df49e219199b8c</td><td><code>keccak256(abi.encodePacked("cEUR", "CELO", "ConstantProduct"))</code></td></tr>
+<tr><td>cREAL/CELO</td><td>0xed0528e42b9ecae538aab34b93813e08de03f8ac4a894b277ef193e67275bbae</td><td><code>keccak256(abi.encodePacked("cREAL", "CELO", "ConstantProduct"))</code></td></tr>
+<tr><td>eXOF/CELO</td><td>0x269dcbdbc07fff1a4aaab9c7c03b3f629cd9bbed49aa0efebab874e4da1ffd07</td><td><code>keccak256(abi.encodePacked("eXOF", "CELO", "ConstantProduct"))</code></td></tr>
+<tr><td>cUSD/axlUSDC</td><td>0x0d739efbfc30f303e8d1976c213b4040850d1af40f174f4169b846f6fd3d2f20</td><td><code>keccak256(abi.encodePacked("cUSD", "axlUSDC", "ConstantSum"))</code></td></tr>
+<tr><td>cEUR/axlUSDC</td><td>0xf418803158d881fda22694067bf6479476cec22ecfeeca2f6a65a6259bdbb9c0</td><td><code>keccak256(abi.encodePacked("cEUR", "axlUSDC", "ConstantSum"))</code></td></tr>
+<tr><td>cREAL/axlUSDC</td><td>0x40c8472edd23f2976b0503db2692e8f06f0eb52db690e84697cad36a6b44e2df</td><td><code>keccak256(abi.encodePacked("cREAL", "axlUSDC", "ConstantSum"))</code></td></tr>
+<tr><td>cEUR/EUROC</td><td>0xfca6d94b46122eb9a4b86cf9d3e1e856fea8a826d0fc26c5baf17c43fbaf0f48</td><td><code>keccak256(abi.encodePacked("cEUR", "axlEUROC", "ConstantSum"))</code></td></tr>
+<tr><td>eXOF/EUROC</td><td>0xcc68743c58a31c4ec3c56bca3d579409b4e2424e5f37e54a85f917b22af74e7c</td><td><code>keccak256(abi.encodePacked("eXOF", "axlEUROC", "ConstantSum"))</code></td></tr>
+<tr><td>cUSD/nativeUSDC</td><td>0xacc988382b66ee5456086643dcfd9a5ca43dd8f428f6ef22503d8b8013bcffd7</td><td><code>keccak256(abi.encodePacked("cUSD", "USDC", "ConstantSum"))</code></td></tr>
+<tr><td>cEUR/nativeUSDC</td><td>0x99be8b8341ba00914600cda701568ab27eea9aca7a32fa48c26e07b86841020c</td><td><code>keccak256(abi.encodePacked("cEUR", "USDC", "ConstantSum"))</code></td></tr>
+<tr><td>cBRL/nativeUSDC</td><td>0xe8693b17c0f002f6a2fe839525557cef10dfeacef9e16c9bbdcb01c57933ce58</td><td><code>keccak256(abi.encodePacked("cREAL", "USDC", "ConstantSum"))</code></td>
+<tr><td>cUSD/USDT</td><td>0x773bcec109cee923b5e04706044fd9d6a5121b1a6a4c059c36fdbe5b845d4e9b</td><td><code>keccak256(abi.encodePacked("cUSD", "USD₮", "ConstantSum"))</code></td>
```

Comment:
I see some weird T here. Not sure how it will show in the actual docs

![Screenshot 2024-06-05 at 8 08 42 PM](https://github.com/mento-protocol/docs/assets/60730266/ec661985-d2f2-4d12-899d-8f0b3a81e307)

Reply:
This T is intended, its used in the USDT contract for the symbol. A normal T would result in the wrong identifier.